src/modules/add-remove.test.js

```diff
@@ -33,4 +34,32 @@ describe('Adding an item', () => {
     removeItem(0, mockTask);
     expect(mockTask.getItem()).toHaveLength(2);
   });
+
+  test('Edit a select task', () => {
+    const index = 0;
+    const desc = "I've been updated";
+    editItem(index, desc, mockTask);
+    expect(todo[index].description).toBe(desc);
+    expect(mockTask.setItem).not.toHaveBeenCalledTimes(1);
+    expect(mockTask.setItem).not.toHaveBeenCalledWith('items', JSON.stringify(todo));
+  });
+
+  test('If task is checked', () => {
+    checkedBox(0, mockTask);
+    expect(todo[0].completed).toBe(true);
+    expect(mockTask.setItem).not.toHaveBeenCalledWith('items', JSON.stringify(todo));
+  });
+
+  test('If task is unchecked', () => {
+    notChecked(0, mockTask);
+    expect(todo[0].completed).toBe(false);
+    expect(mockTask.setItem).not.toHaveBeenCalledWith('items', JSON.stringify(todo));
+  });
+
+  test('Clear all unchecked tasks', () => {
+    clearTasks();
+    for (let index = 0; index < mockTask.length; index += 1) {
+      expect(JSON.parse(mockTask.setItem.mock.calls[index]).toEqual([{ completed: false }]));
+    }
+  });
```

Comment:
- [ ] Great job on implementing the test as specified in the project requirement! 👏 However, please note that you have to create a new test file for each of the functionalities you want to test as mentioned in the project requirements instead of writing all your tests in one file. See below for more details on project requirements:
 
<img width="678" alt="Screenshot 2023-07-20 at 17 14 52" src="https://github.com/Arch-Noize/to-do-list/assets/107192076/efa60a21-a1b4-4147-a795-59dc16ef84e2">